docs/doxygen/modIngest.dox

```diff
@@ -198,27 +198,23 @@ See \ref platform_blackboard  for details on posting results to it.  You use the
 The blackboard defines artifacts for specific data types (such as web bookmarks). 
 You can use one of the standard artifact types or create your own.
 
-When modules add data to the blackboard, they should notify listeners of the new 
-data by invoking the org.sleuthkit.autopsy.ingest.IngestServices.fireModuleDataEvent() method. 
-Do so as soon as you have added an artifact to the blackboard.
-This allows other modules (and the main UI) to know when to query the blackboard 
-for the latest data.  However, if you are writing a large number of blackboard 
-artifacts in a loop, it is better to invoke org.sleuthkit.autopsy.ingest.IngestServices.fireModuleDataEvent() 
-only once after the bulk write, so as not to flood the system with events.
-
-Further, when modules create artifacts, they should be indexed for keyword search,
-using the method org.sleuthkit.autopsy.casemodule.services.Blackboard.indexArtifact(BlackboardArtifact artifact). This can be done
-in the following way:
-
-\code
-Blackboard blackboard = Case.getCurrentCase().getServices().getBlackboard();
-try {
-	blackboard.indexArtifact(artifact); //Your artifact as the argument.
-}
-catch (BlackboardException ex) {
-	//YOUR EXCEPTION BEHAVIOR HERE.
-}
-\endcode
+After you make an artifact, you should call <a href="http://sleuthkit.org/sleuthkit/docs/jni-docs/4.6/classorg_1_1sleuthkit_1_1datamodel_1_1_blackboard.html">sleuthkit.Blackboard.postArtifact()</a>, which will:
+<ul>
+<li>Analyze the artifact and add any timestamps to the Timeline tables
+<li>Send an event over the EventBus that the artifact(s) was added
+<ul>
+<li>Autopsy is a listener of this EventBus and will rebroadcast it to other Autopsy modules
+<li>Keyword search will also listen for this and will index the artifact
+</ul>
+</ul>
+
+This means you no longer have to make calls to:
+ - Index the artifact
+ - Fire the event to refresh the UI.
+ 
+If you are creating a large number of artifacts, you may see better performance if you save all the artifacts and do one bulk write at the end using <a href="http://sleuthkit.org/sleuthkit/docs/jni-docs/4.6/classorg_1_1sleuthkit_1_1datamodel_1_1_blackboard.html">sleuthkit.Blackboard.postArtifacts()</a>
+ 
```

Comment:
I think it would be helpful to mention that we can batch making the artifacts as well as just "posting" them. I find "posting" to be unfortunately ambiguous - to me it sounds like that method would also add the artifacts.

Reply:
We can make artifacts in batches? I know we can batch up the attributes but I didn't think you could make an artifact without adding it to the DB at the same time. Regardless, the wording is a bit confusing (in the top paragraph it does use "post" to mean "add to DB") so I'll work on that.

Reply:
No, it turns out that we cannot make artifacts in batches. It's attributes that we can make in batches. In fact, BlackboardArtifact does not have a public constructor to allow such a thing. Maybe we should change this and provide a way to add artifacts in batches, but that is a story for another day. Sorry for my confusion.